### ISSUE TITLE
Approving a translation made me a contributor

### ISSUE THREAD

**Issue by FArekkusu** [Codewars/codewars.com#1718]
I'm listed as a contributor [here](https://www.codewars.com/kata/closest-pair-of-points-in-linearithmic-time) _presumably_ after I approved a Java translation. I neither wrote any translations myself nor opened the kata editor (IIRC).

┆Issue is synchronized with this [Clickup ](https://app.clickup.com/t/2me1c84) by [Unito](https://www.unito.io)

**Comment by FArekkusu** [comment]
Just happened [here](https://www.codewars.com/kata/57a1ae8c7cb1f31e4e000130) and [here](https://www.codewars.com/kata/5a58d46cfd56cb4e8600009d). I'm listed as a contributor now, and on the revisions page it says that the changed made to the description by the translator were made by me. Perhaps, when somebody approves a translation with modified description, they become a contributor too.

**Comment by error256** [comment]
> Perhaps, when somebody approves a translation with modified description, they become a contributor too.

Yes, it looks like this. I'm not sure if it happens every time when the description is modified, but I think it never happens when there are no changes.
